modules/core/src/main/java/org/apache/ignite/configuration/IgniteConfiguration.java

```diff
@@ -2678,11 +2678,13 @@ public IgniteConfiguration setAutoActivationEnabled(boolean autoActivation) {
      * Gets state of cluster on start.
      * <br/>
      * For <b>in-memory cluster</b> this state will be applied to the first started node. If
-     * cluster state on start is {@link ClusterState#INACTIVE}, there will be no cache partition map exchanges
-     * performed until the cluster is activated. This should significantly speed up large topology startup time.
+     * cluster state on start is {@link ClusterState#INACTIVE}, further hode joins will be handled by cluster faster and
+     * manual cluster activation should be performed in order to start working the cluster and caches.
      * <br/>
      * For <b>persistent cluster</b> If state is differ from {@link ClusterState#INACTIVE} and BaselineTopology is set
```

Comment:
state is different from / state differs from

Reply:
done